change/@fluentui-react-components-33328e56-76f3-4502-8a64-2417781aa89c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27341&commitId=e6460553086614faa7e4ee762eb8b5c2f8850c35&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1902 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Fluid.normal.chromium.png|361197| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>